### DISCUSSION TITLE
"failed to solve: process "/bin/sh -c pip install  --cache-dir=/var/cache/pip  --target=lib -r requirements.txt" when trying to run dev locally with chat

### DISCUSSION THREAD

**Discussion by samy-rashed03** [LAION-AI/Open-Assistant#3349] [Q&A]
I was able to get OpenAssistant running locally *without* chat just fine, however I'm encountering main issues when trying to run locally with chat. I was able to switch Dockerfile.worker-full to Dockerfile.worker-standalone in the docker compose config, but I'm still getting issues. The attachment to this issue can hopefully outline my problems
<img width="573" alt="image" src="https://github.com/LAION-AI/Open-Assistant/assets/56412118/67339b61-e267-4876-9131-874314bbb717">

If anyone can help me, it'd be great. I'm running OpenAssistant on an M1 MacBook and the platform in my docker compose config is "linux/x86_64".

**Comment by AbdBarho** [comment]
can you try again with `docker compose --profile inference --progress plain`? this would show the error better.

In any case, it seems that one of the libraries do not exist for arm / M1 (and probably M2)
